ajax/libs/phaser-ce/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "phaser-ce",
+  "version": "2.7.9",
+  "description": "Phaser CE (Community Edition) is a fast, free and fun HTML5 Game Framework for Desktop and Mobile web browsers.",
+  "filename": "phaser.min.js",
+  "author": {
+    "name": "Richard Davey",
+    "email": "rdavey@gmail.com",
+    "url": "http://www.photonstorm.com"
```

Comment:
@extend1994 I think we don't use https
because the url always redirect to 80 port (http)